pyproject.toml

```diff
@@ -98,6 +99,7 @@ base = [
     "openvino-model-api==0.2.4",
     "onnx==1.17.0",
     "onnxconverter-common==1.14.0",
+    "onnxruntime==1.20.0",
```

Comment:
Why do we need onnxruntime in basic OTX installation? It seems like it's only used in tests

Reply:
This is to ensure that ONNX model can be successfully loaded. Most detection models work with ONNX, so it makes sense to test it since we support ONNX export. Would you recommend adding an ONNX inference tests as well?

Reply:
I see, but in general adding tests deps to the base installation is questionable. Could you move onnxrt to the test deps section?